lib/plugins/layer2/strike/strike.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   cryptoNetwork
 }
 
-axios.defaults.baseURL = options.strike.baseUrl
+axios.defaults.baseURL = _.toString(_.get(options, 'strike.baseUrl', ''))
```

Comment:
How does it help to default to blank baseURL? Should probably throw an error, instead.